.travis.yml

```diff
@@ -1,39 +0,0 @@
-sudo: required
-dist: trusty
-
-addons:
-  apt:
-    sources:
-      - chef-current-trusty
-    packages:
-      - chefdk
-
-# Don't `bundle install` which takes about 1.5 mins
-install: echo "skip bundle install"
-
-branches:
-  only:
-    - master
-
-services: docker
-
-env:
-  matrix:
-  - INSTANCE=default-centos-6
-  - INSTANCE=default-centos-7
-  - INSTANCE=default-ubuntu-1604
-
-before_script:
-  - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
-  - eval "$(chef shell-init bash)"
-  - chef --version
-  - cookstyle --version
-  - foodcritic --version
-
-script: KITCHEN_LOCAL_YAML=kitchen.dokken.yml kitchen verify ${INSTANCE}
-
-matrix:
-  include:
-    - script:
-      - chef exec delivery local all
-      env: UNIT_AND_LINT=1
```

Comment:
We'll need to delete the webhook

Reply:
How do we do that? I have added a terraform file, will that do it?

Reply:
It's on the settings tab... Which you don't have 🤦‍♂️because you're not on the maintainers